code/_onclick/hud/parallax.dm

```diff
@@ -4,6 +4,8 @@
 	var/client/C = screenmob.client
 	if (!apply_parallax_pref(viewmob)) //don't want shit computers to crash when specing someone with insane parallax, so use the viewer's pref
 		return
+	if (SSmapping.level_trait(viewmob.z, ZTRAIT_NOPARALLAX))
+		return
```

Comment:
you shouldn't need this, I was wrong. the check inside apply_parallax_pref ought to catch this case